PurpleCoin/PurpleCoin/Presentation/Market/Cell/CoinTableViewCell.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CoinTableViewCell.swift
+//  PurpleCoin
+//
+//  Created by 강재혁 on 2023/12/24.
+//
+
+import UIKit
+
+class CoinTableViewCell: UITableViewCell {
+    
+    let koreanCoinNameLabel: UILabel = {
+        let label = UILabel()
+        label.text = "도지코인"
+        label.textColor = .white
+        label.font = PurpleCoinFont.font(type: .medium, size: 12)
+        return label
+    }()
+    let englishCoinNameLabel: UILabel = {
+        let label = UILabel()
+        label.text = "DOGE/KRW"
+        label.textColor = PurpleCoinColor.gray
+        label.font = PurpleCoinFont.font(type: .medium, size: 10)
+        return label
+    }()
+    let currentPriceLabel: UILabel = {
+        let label = UILabel()
+        label.text = "135"
+        label.textColor = .white
+        label.font = PurpleCoinFont.font(type: .medium, size: 12)
```

Comment:
```suggestion
        label.font = PurpleCoinFont.font(type: .medium, size: 12)
```

별도의 함수나 객체로 분리해주시기 바랍니다.